packages/tpetra/core/src/Tpetra_FECrsGraph_decl.hpp

```diff
@@ -252,16 +252,7 @@ namespace Tpetra {
     //! @name Collective methods for changing the graph's global state
     //@{
 
-    /// \brief Communicate nonlocal contributions to other processes.
-    ///
-    /// This method is called automatically by fillComplete().  Most
-    /// users do not need to call this themselves.
-    ///
-    /// This method must be called collectively (that is, like any MPI
-    /// collective) over all processes in the graph's communicator.
-    void globalAssemble (){endFill();}
```

Comment:
If you removed this due to an Intel compiler bug, you could try qualifying the method with `this->`.